### DISCUSSION TITLE
10th

### DISCUSSION THREAD

**Discussion by Noroz-77** [avelino/awesome-go#5264] [Q&A] [answered]
### A third-level heading

**Comment by Noroz-77** [comment]
Hi no problem?

**Reply by Noroz-77** [reply]
Hello, I really apologize to you. I am a young person who is new to this business and I also work in a cafe and the cost of living is very high. Please help me so that I can have a good extraction.

**Comment by phanirithvij** [comment, marked as answer]
If there is a problem please give a screenshot or a link or send a PR. What do you expect 10th means to anyone looking at it.

**Reply by phanirithvij** [reply]
Also you are a likely spammer given that your account was created less than a day ago and you are following me the only one who interacted with you on GitHub. If you are not spamming try and convince me.

**Reply by Noroz-77** [reply]
Hello, I really apologize to you. I am a young person who is new to this business and I also work in a cafe and the cost of living is very high. Please help me so that I can have a good extraction.

**Reply by phanirithvij** [reply]
I can't help you

**Reply by phanirithvij** [reply]
I don't know how to help with extraction, hope you get extracted.
